docs/GettingStarted.md

```diff
@@ -1369,6 +1371,36 @@ If you wish to configure each connection object individually, you may use the `D
 client = Net::HTTP.new(host, port)
 Datadog.configure_onto(client, **options)
 ```
+### Postgres
+
+The PG integration traces SQL commands sent through the `pg` gem via `exec`, `exec_params`, `exec_preapred`, `async_exec`, `async_exec_params`, `async_exec_prepared`,`sync_exec`,`sync_exec_params` or `sync_exec_prepared`.
+
+```ruby
+require 'pg'
+require 'ddtrace'
+
+Datadog.configure do |c|
+  c.tracing.instrument :pg, options
+end
+
+conn = PG.connect( :dbname => 'database', :host => 'db', :user => 'user', :password => 'password' )
+conn.exec('SELECT * FROM users WHERE group="x"')
+conn.exec_params('SELECT * FROM users WHERE group=$1', ['x'])
+# conn.prepare('FIND USER BY GROUP', 'SELECT * FROM users WHERE group=$1;')
+conn.exec_prepared('FIND USER BY GROUP', ['x'])
+conn.async_exec('SELECT * FROM users WHERE group="x"')
+conn.async_exec_params('SELECT * FROM users WHERE group=$1', ['x'])
+conn.async_exec_prepared('FIND USER BY GROUP', ['x'])
+conn.sync_exec('SELECT * FROM users WHERE group="x"')
+conn.sync_exec_params('SELECT * FROM users WHERE group=$1', ['x'])
+conn.sync_exec_prepared('FIND USER BY GROUP', ['x'])
```

Comment:
We normally don't document user-code examples here, only configuration examples.
Sometimes user-code is relevant for configuration, but I don't really see it in this case.

Besides, you already did a great job at document all the supported PG methods in the paragraph above this code snippet, so I'm not sure these examples here are needed.